### DISCUSSION TITLE
VSCode extension: color display

### DISCUSSION THREAD

**Discussion by Faithfinder** [chakra-ui/panda#2983] [Ideas]
Add an inline adornment with color, similar to what Color Highlight extension does for plain colors

**Comment by anubra266** [comment]
@Faithfinder https://marketplace.visualstudio.com/items?itemName=chakra-ui.panda-css-vscode

**Reply by Faithfinder** [reply]
Yes, that is the extension I'm talking about =P

What I meant is, see in the screenshot how `borderColor: "#f00"` is highlighted in, well, `#f00`, via [Color Highlight](https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight) extension,  but `color: "fuchsia"` is not, even though hover shows that Panda extension does know what the token resolves to?
<img width="230" alt="image" src="https://github.com/user-attachments/assets/abbaa72f-1e54-417e-8408-854527b8d901">
